pyvast/example/example.py

```diff
@@ -12,7 +12,9 @@
 Prerequisites: VAST must be running.
     0. Install VAST
     1. Go to another terminal
-    2. `vast start`
+    2. `vast start` (start a vast node)
+    3. Go to another terminal
+    4. `python -m pip install -r requirements.txt` (install requirements)
```

Comment:
You have committed a `Makefile`, would it make sense to reference it here?

Reply:
The `requirements.txt` are specifically meant for the example. They install `pyarrow` and `pandas` to make the example work. The `Makefile` facilitates the build and dist setup for the PyVAST project itself and is not really needed here.